tests/builder/axes_test.py

```diff
@@ -347,8 +351,60 @@ def test_axis_mapping(ufo_module):
     assert doc.axes[1].map != wdth_mapping
     assert doc.axes[1].map == []
 
-    # When we convert back to glyphs, the wdth mapping isn't present because
-    # it was removed during the designspace conversion.
-    font = to_glyphs(doc)
-    axis_mappings["wdth"] = {}
-    assert font.customParameters["Axis Mappings"] == axis_mappings
+    font2 = to_glyphs(doc)
+    assert font2.customParameters["Axis Mappings"] == axis_mappings
```

Comment:
Rename font2 back to font as it was originally (as in suggestion below) 
```suggestion
    font = to_glyphs(doc)
    assert font.customParameters["Axis Mappings"] == axis_mappings
```